src/enums/componentsCompanyInfo.ts

```diff
@@ -15,5 +15,6 @@ export enum ComponentsCompanyInfo {
   'people-and-roles',
   'share-structures',
   'company-provisions',
-  'resolution-dates'
+  'resolution-dates',
+  'association-type'
```

Comment:
As per requested in 12895 - Needed this, so it will scroll back to this component

Reply:
Order is import (so it scrolls to the first invalid component). Does association-type come after resolution-dates?

Reply:
Yes - I'll double check

Reply:
Yes